specifyweb/frontend/js_src/lib/components/WbUtils/index.tsx

```diff
@@ -46,8 +50,12 @@ export function WbUtilsComponent({
             title={commonText.searchQuery()}
             type="search"
             onKeyDown={_.debounce(
```

Comment:
hmm, I don't think debounce would work correctly if you are creating a new debounce function on each render - you might need to create debounce inside of useMemo instead